package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "type-check": "tsc --noEmit",
     "type-check:watch": "npm run type-check -- --watch",
-    "build:typescript": "rimraf ./@types && rimraf ./lib && tsc",
-    "build:webpack": "rimraf ./out && webpack --mode=production",
+    "build:typescript": "tsc",
```

Comment:
Getting rid of `rimraf` calls makes npm linking in the product repo smoother, because webpack does not crash as a result of dependency files being deleted.

Reply:
This means you can run `npm run build:watch` in model.js, make changes, and things will be automatically recompiled all the way down to the product dev server.

Reply:
Nice! I suspect I was using `rimraf` to account for deleted or renamed source files, which would have been much more likely to happen during early development, but not so much anymore.